idseq_dag/steps/run_alignment_remotely.py

```diff
@@ -360,6 +363,16 @@ def __copy_multihit_remote_outfile(self, key_path, remote_username, instance_ip,
             )
         )
 
+    @command.retry
+    def __delete_remote_dir(self, remote_dir, key_path, remote_username, instance_ip):
+        """
+        Delete a directory on a remote machine
+        This needs to happen while we are holding the machine reservation,
+        i.e., inside the "with ASGInstnace" context.
+        """
+        rm_command = f"rm -r {remote_dir}"
```

Comment:
Shouldn't it be `rm -rf` for extra robustness in case the directory failed to get created in the first place?   Minor point, and probably doesn't matter in this case, but generally speaking when I see `rm -r` without `rm -rf` it raises a flag and makes me wonder did the code depend on an exception getting raised if the path didn't exist.